src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    private static final String BANANA = "banana";
+    private static final String APPLE = "apple";
```

Comment:
Consider removing the hardcoded fruit types ('banana' and 'apple') to make the application more flexible. Instead, you could validate against a list of allowed fruits that can be configured or extended as needed.